src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -172,3 +172,9 @@ def _lock_cb(cls, mode, n, file, line):
                     mode, n, file, line
                 )
             )
+
+
+# OpenSSL is not thread safe until the locks are initialized. We initialize in
+# module scope to cause initialization whenever this module is imported (and
+# try to get some benefit from the import lock).
```

Comment:
Can you both shorten and expand here "We initialize at module scope so that we run with the import lock [explain why the import lock is relevant]"